Automattic/jetpack
# Moved plan-utils to shared-extension-utils package

Fixes #

#### Changes proposed in this Pull Request:

* Moved `requiredPaidPlan` method from `register-jetpack-block` file to `plan-utils` file
* Moved `plan-utils` from jetpack plugin folder to `shared-extension-utils` package and updated all import references

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?
No



#### Testing instructions:
* Make sure CI is happy
* `pnpm build-extensions` command should run without any errors in Jetpack plugin folder
* Test if proper upgrade URL is injected:

1. Go to your Jurassic tube site. Make sure it doesn't have any active plan.
2. Open block editor and add "Donations Form" block or any similar premium block.
3. Save the draft and preview in the new tab. You should see a nudge like in the image below and generated Upgrade URL should take you to the relevant plan buy page
<img width="816" alt="Screenshot 2022-03-09 at 3 21 09 AM" src="https://user-images.githubusercontent.com/95693514/157331585-ddc4224a-5cf0-49d3-945a-09203f53fa9e.png">








* Go to '..'
*
